core/unit_test/CMakeLists.txt

```diff
@@ -336,9 +336,9 @@ KOKKOS_ADD_ADVANCED_TEST( UnitTest_PushFinalizeHook_terminate
 )
 
 if(Kokkos_ENABLE_PROFILING)
-  add_library(
-    printer-tool MODULE
-    tools/printing-tool.cpp
+  KOKKOS_ADD_LIBRARY(
```

Comment:
@jjwilke : is this going to cause problems elsewhere? I seem to remember the MODULE formulation being preferred

Reply:
I changed this since `TRIBITS` doesn't seem to allow `MODULE`.

Reply:
You should have commented :/

Reply:
This change seems to be causing the library to be installed with Kokkos